backend/submissions/utils/corpus_processor.py

```
@@ -0,0 +1,21 @@
+import PyPDF2
+import os
```

Ruff (0.11.9)

`os` imported but unused

Remove unused import: `os`

(F401)